src/cards/ha-weather-card.html

```diff
@@ -14,23 +14,29 @@
         }
 
         .attribution {
-          color:grey;
+          color: var(--secondary-text-color);
+          text-align: right;
```

Comment:
There are a couple of ways of making it reusable, the 'easiest' would be to create a CSS mixin that can be applied in cards. https://www.polymer-project.org/1.0/docs/devguide/styling#custom-css-mixins